src/css/components/hexagon.scss

```diff
@@ -0,0 +1,47 @@
+
+$primary-color: $color-primary-alt;
+$alt-color: $color-secondary;
+
+.hexagon {
+	background: $primary-color;
+  height: .65em;
+	position: relative;
+  width: 1em;
```

Comment:
These are actually all supposed to be `em` because you can set teh font-size of the parent to control the size of the hexagon. This was done rather then using percentage units for everything because border sizes can't be defined with percentages but can be defined with `em`s.